### DISCUSSION TITLE
NC "metronome backtrack" should be disable-able like the metronome in Muted

### DISCUSSION THREAD

**Discussion by zachmanthethird** [ppy/osu#16727] [Ideas]
When playing NC, there is a "backtrack" that acts as a metronome. This isn't currently disable-able, but when applying the Muted mod, it defeats the purpose as it won't be muted. I suggest that the NC "backtrack" should be affected by the Muted mod if it doesn't receive its own toggle.
![image](https://user-images.githubusercontent.com/74262407/151920435-b9dab5c3-e772-435e-a0c0-7c139cdb00fe.png)
(Edit: Semi-duplicate of #14209)

**Comment by peppy** [comment]
I dunno, it's half the point of nightcore though.

**Reply by uzervlad** [reply]
There is a reason to why skinners have been removing those sounds. They can sometimes be very annoying and/or distracting.

Also, for no reason whatsoever, some maps just feel easier with higher pitch and no nightcore beats.

**Reply by mk56-spn** [reply]
^^ many of us NC mains play for the undistorted audio or pitched vocals

**Reply by peppy** [reply]
Leave it to skins to undo core game mechanics IMO.

**Reply by UntitledPhysicist** [reply]
I did attempt to remove the nightcore backtrack via a skin and it worked in osu! classic but for some reason it did not work in lazer

**Reply by UntitledPhysicist** [reply]
So, I got it to work, I don't know if this is how it is intended or not but if you update a skin that was imported to lazer (specifically modifying the files) the updates made are reflected in osu classic but not lazer. I updated the files and reimported it into lazer and now it is working.

**Comment by frenzibyte** [comment]
I don't understand what this is discussing about. The title states that the nightcore beats should be disable-able, which is a clear duplicate of #14209. While the description states that... the "Muted" mod should affect the nightcore beats?

I would argue it shouldn't as the nightcore beats are not part of either the track or the hitsounds which is what `ModMuted` should mute. If anything, I think the opposite should happen, i.e. `ModNightcore` should check if a metronome was added already and mutes it completely to not interfere with the nightcore beats. Nightcore beats should not defeat the point as it makes sense to be played while the track is both muted or unmuted.

As for making the nightcore beats disable-able in general, that's out of scope from this discussion and should continue in #14209 instead.

**Reply by zachmanthethird** [reply]
I guess it boils down to whether DoubleTime and NightCore have the same difficulty. I assumed that, like stable, they would be equal in difficulty. However, if the backtrack is not disable-able, this cannot be true. NC+Muted would be objectively much easier than DT+Muted.

**Reply by frenzibyte** [reply]
It would be objectively easier when you disable the metronome in "Muted", sure. I suppose that case needs better handling.
